parser/parser_test.go

```diff
@@ -727,6 +727,9 @@ func (s *testParserSuite) TestIdentifier(c *C) {
 		{`select 1 a, 1 "a", 1 'a'`, true},
 		{`select * from t as "a"`, false},
 		{`select * from t a`, true},
+		// reserved keyword can't be used as identifier directly, but A.B pattern is an exception
+		{`select COUNT from DESC`, false},
+		{`select COUNT from SELECT.DESC`, true},
```

Comment:
Could you add unit test like: https://github.com/pingcap/tidb/blob/master/parser/parser_test.go#L55
So all the keywords will be covered.